### PR TITLE
UiPath Automation Cloud [v2 features]

### PR DESCRIPTION
**Pull request by TuckermanRav**
- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

The purpose/goal of this PR is to introduce V2 of our currently published connector "UiPath Orchestrator".  We are choosing to publish a net new connector as we are introducing:

- Connector name change and alignment with evolution of product name.
- New authentication mechanism,
- Additional features which rely on mentioned auth mechanism.

We plan to deprecate the existing v1 connector entirely after this connector is published.

### CODE REVIEW

File: certified-connectors/UiPath Automation Cloud/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Run a job in UiPath Orchestrator
+- [Add Queue Items](https://docs.uipath.com/orchestrator/docs/about-queues-and-transactions)
+Add a queue item to a specific queue
+
+## Getting Help or Providing Feedback
+If you have any issues, requests for functionality, or have general feedback, please <integrations-apps@uipath.com>.
+
+## Pre-requisites
+Please make sure you have an automation cloud account. [Sign up](https://www.uipath.com/developers/studio-download) for a trial.
+
```
Comment (vmanoharas) [inline certified-connectors/UiPath Automation Cloud/ReadMe.md]: Hello @TuckermanRav, 

Please update readme.md file as per the template here and fill all the required fields. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by vmanoharas — COMMENTED** [review]
Please resolve review comments.

File: certified-connectors/UiPath Automation Cloud/ReadMe.md (hunk 1 above)
Comment (TuckermanRav) [reply certified-connectors/UiPath Automation Cloud/ReadMe.md]: Thanks @vmanoharas.  We've addressed the readme per the template now.

Comment (vmanoharas) [reply certified-connectors/UiPath Automation Cloud/ReadMe.md]: Hello @TuckermanRav, 

The readme.md sections are not as per the template [https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md].  Please refer other certified connectors readme file and follow the same. For instance - https://github.com/microsoft/PowerPlatformConnectors/pull/2948/files#diff-ca3429f2aa84e4528cbfd6c234d4a96576947c9cf8c86524d267abc942d335da

**Review by vmanoharas — COMMENTED** [review]
Please resolve the readme file review comments.

File: certified-connectors/UiPath Automation Cloud/ReadMe.md (hunk 1 above)
Comment (TuckermanRav) [reply certified-connectors/UiPath Automation Cloud/ReadMe.md]: @vmanoharas I believe we have resolved all readme issues.

**Review by vmanoharas — APPROVED** [review]
Hi @TuckermanRav, 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**